README.rst

```diff
@@ -589,6 +589,87 @@ Django setting ``DJANGO_REDIS_CONNECTION_FACTORY``.
             # creating a new connection using the `get_connection` method.
             pass
 
+Use the sentinel connection factory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to facilitate using `Redis Sentinels`_, django-redis comes with a
+built in sentinel connection factory, which creates sentinel connection pools.
+In order to enable this functionality you should add the following:
+
+
+.. code-block:: python
+    :caption: myproj/settings.py
```

Comment:
Looking at https://github.com/github/markup it looks like this https://github.com/github/markup/blob/master/lib/github/commands/rest2html script is responsible, but it might be docutils' fault in the end (I haven't looked that far)

Reply:
I opened https://github.com/github/markup/pull/1452 to hopefully fix this. If not we should remove the Sphinx directives since I don't think any documentation is built with Sphinx anymore.